addons/mail/static/src/components/composer/composer.scss

```diff
@@ -161,6 +158,14 @@
 // Style
 // ------------------------------------------------------------------
 
+// TODO FIXME o-open on the button should be enough.
+// Style of button when popover is "open" comes from web.Popover, and we can't
+// define a modifier on .o_Composer_button due to not being aware of Popover's
+// state in context of template. https://github.com/odoo/owl/issues/693
+.o_is_open .o_Composer_button {
+    background-color: gray('200');
+}
```

Comment:
I think we intent to have this rule only for `o_Composer_buttonEmojis` and not for any other button (especially not send button and such where is_open doesn't make sense), or maybe just `o_Composer_toolButton`?

Reply:
Same for `&.o-open` currently in `.o_Composer_button`